lib/tasks/generate_2fa_encription_key.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+namespace :generate do
+  desc "Generates a key for encrypting 2fa tokens"
+  task :twofa_key do
```

Comment:
I guess we should automagically call this akin to how we do it for the secret token, also have handling for it for ephemeral filesystems such as heroku's. Have a peek in `lib/configuration_methods.rb` `secret_token` method and its usages.

Reply:
I was wondering about that :D I basically copied and adjusted the secret_token method. I hope it works like that